CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.6
+
+- Add `isAssertionError` type matcher.
```

Comment:
Testing assertion behavior feels a little iffy to me - but I suppose it could make sense if you are writing tests for a framework?

Reply:
We had a case in devtools where this was necessary. The corresponding CL fixed a bug and we wanted to make sure we'd continue to catch it via assertion in the future. https://github.com/flutter/devtools/blob/master/packages/devtools/test/cpu_profile_protocol_test.dart#L35